consensus/consensus.go

```diff
@@ -44,7 +44,7 @@ type Consensus struct {
 
 // NewConsensus creates a MOCA consensus
 func NewConsensus(account *vault.Account, localNode *node.LocalNode) (*Consensus, error) {
-	txnCollector := chain.NewTxnCollector(localNode.GetTxnPool(), config.MaxNumTxnPerBlock)
+	txnCollector := chain.NewTxnCollector(localNode.GetTxnPool(), int(config.Parameters.MaxNumTxnPerBlock))
```

Comment:
It is easy to confuse between  config.MaxNumTxnPerBlock with config.Parameters.MaxNumTxnPerBlock

Reply:
Agreed, renamed to NumTxnPerBlock